src/sentry/ingest/ingest_consumer.py

```diff
@@ -143,12 +143,13 @@ def _do_process_event(message, projects):
         logger.error("Project for ingested event does not exist: %s", project_id)
         return
 
-    # Parse the JSON payload. This is required to compute the cache key and
-    # call process_event. The payload will be put into Kafka raw, to avoid
-    # serializing it again.
-    # XXX: Do not use CanonicalKeyDict here. This may break preprocess_event
-    # which assumes that data passed in is a raw dictionary.
-    data = json.loads(payload)
+    with sentry_sdk.start_span(op="ingest_consumer.process_event.parse_payload"):
+        # Parse the JSON payload. This is required to compute the cache key and
+        # call process_event. The payload will be put into Kafka raw, to avoid
+        # serializing it again.
+        # XXX: Do not use CanonicalKeyDict here. This may break preprocess_event
+        # which assumes that data passed in is a raw dictionary.
+        data = json.loads(payload)
```

Comment:
With https://github.com/getsentry/sentry/pull/25541/files#diff-5c1440eab19b478087e2a08ff5238d26b56c916cc7baa1b481a9645cab9200abR109, this will be covered by two spans, right?

Reply:
wups